Firestore/CHANGELOG.md

```diff
@@ -18,8 +18,8 @@
   `setData(["a": "b"], merge: true)`.
 - [changed] Replaced the `SnapshotOptions` object with direct use of the
   `FIRServerTimestampBehavior` on `DocumentSnapshot`. Instead of calling
-  `get(field, SnapshotOptions.serverTimestampBehavior(.estimate))` call
-  `get(field, serverTimestampBehavior: .estimate)`.
+  `data(SnapshotOptions.serverTimestampBehavior(.estimate)` call
```

Comment:
You're missing a close-paren, and I think it should be`data(with:...` no?

Reply:
Yes. Also fixed the spelling on "Changed" in the line below.